### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by justjosias** [muesli/go-app-paths#5]
I am unsure if this is within scope for the project or not, but for my project Android support would be very useful. `/data/data/<app_id>` and `Android/data/` are among the options.

**Comment by muesli** [comment]
Yes, I'd certainly be interested to support more platforms! I must admit that I have to read up on the various Android versions and their default storage locations. If you have any useful pointers or insight there, I'd greatly appreciate it!
